### PR TITLE
Branch for added features

### PR DESCRIPTION
**Pull request by ClintonYounge**
Added-

1. All code necessary to implement a responsive home and about page.
2. A mobile menu that'll allow the visitor to switch between pages.
3. A section of the home page made dynamically with JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,79 +8,140 @@
...
+- A guides section made dynamically with javascript.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Below is a link to the Live Demo of my project.
+- [Experience Guyana](coming soon)
```
Comment (zieeco) [inline README.md]: - Let's kindly deploy the this website using Github pages to meet this part of the project requirement.

File: README.md [hunk 2]
```diff
@@ -6,79 +8,140 @@
...
+
+> Below is a link to Loom explaining my project.
+- [My Loom Video](coming soon.)
```
Comment (zieeco) [inline README.md:62]: - Please, let's kindly add the presentation video. Please, check the project requirement to follow the instructions regarding adding the video to meet this part of the project requirement.

File: README.md [hunk 3]
```diff
@@ -6,79 +8,140 @@
...
-Give a ⭐️ if you like this project!
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+> If you like this project please give me a star on GitHub.
+<!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank my standup team for help settling my mind to persevere through this project.
+> Many thanks to [Microverse](https://www.microverse.org/) for teaching me all I know about programming.
```
Comment (zieeco) [inline README.md]: - Let's also give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to meet this part of the project requirement.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+                    <div class="event-date">
+                        <p>2023.02.23 (THUR)</p>
+                    </div>
+                    <div class="event-venue">
+                        <p>@ Guyana National Park (Georgetown)</p>
+                    </div>
+                </section>
+            </div>
+        </section>
+    </section>
```
Comment (zieeco) [inline index.html]: ### THIS CHANGE IS APPLICABLE TO THE ABOUT PAGE ALSO

- Hey, Please, let's kindly improve our page structure by doing the following to enforce a better way of having a semantic index page below:

- [ ] Kindly remove the <header> from this section as the header hold the introductory contents of your website so it should be a stand-alone element not wrapped in a section. also, the header should be the first element in the body before another element of your page.
- [ ] The   `<div class="social">` should be moved into the nav inside the header as this is a nav item.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+                    <box-icon name='facebook-circle' type='logo'></box-icon>
+                </li>
+                <li>
+                    <box-icon type='logo' name='twitter'></box-icon>
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
```
Comment (zieeco) [inline index.html]: ### THIS CHANGE IS APPLICABLE TO THE ABOUT PAGE ALSO


- Please, let's follow the design properly paying attention to the background color, and the colors of the icons.

*design*
Screenshot from 2023-02-16 22-04-52

*your design*
Screenshot from 2023-02-16 22-04-26

File: index.html [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+        
+        <header class="nav-header">
+            <nav class="navbar">
+                <div class="menu-icon">
+                    <box-icon name='menu' size="lg" class=".hamburger"></box-icon>
+                </div>
+                <li class="logo"><img src="./images/left-logo.png" alt=""></li>
```
Comment (zieeco) [inline index.html]: - Please, kindly use another element like a div to wrap the logo as the li is ideal to be used with the ul as it signifies a list item. using the li alone is not semantically correct.

File: about-page/about.html [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+                <li>Logout</li>
+            </ul>
+        </div>
+        
+        <header class="about-nav-header">
+            <nav class="about-navbar">
+                <div class="about-menu-icon">
+                    <box-icon name='menu' size="lg" class=".hamburger"></box-icon>
+                </div>
+                <li class="logo"><img src="images/left-logo.png" alt=""></li>
```
Comment (zieeco) [inline about-page/about.html]: - Please, kindly use another element like a div to wrap the logo as the li is ideal to be used with the ul as it signifies a list item. using the li alone is not semantically correct.
- Also, the logo is not rendering, kindly fix it.

File: about-page/about.html [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+            <p class="tail">Take a look at the last two events we hosted</p>
+        </div>
+        <div class="past-event-images">
+            <div class="image1">
+                <a href="#"><img src="/images/Experience-Guyana-Logo2.png" alt="Experience Guyana previous year logo"></a>
+            </div>
+            <div class="image2">
+                <a href="#"><img src="/images/Experience-Guyana-Logo3.png" alt="Experience Guyana previous year logo"></a>
+            </div>
+        </div>
```
Comment (zieeco) [inline about-page/about.html]: - Hey, Please, let's improve our design on each card to match the design as much as we can.
   - The date should come first before the next content
   - Improve the color of each card
   - The date and content have a white color.
   - see the image below to compare your design


*design*
Screenshot from 2023-02-16 22-19-06

*your design*
Screenshot from 2023-02-16 22-18-43

File: about-page/about.html [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+        <div class="footer-bottom">
+            <div class="bottom-left">
+                <p>experience guyana <br>come 23rd february, 2023</p>
+            </div>
+            <div class="bottom-right">
+                <p>2023, some rights reserved</p>
+            </div>
+        </div>
+    </footer>
```
Comment (zieeco) [inline about-page/about.html]: ## THIS CHANGE IS APPLICABLE TO THE HOME PAGE `index.html` 

- Let's also improve on the footer, the highlighted part of the image below should be the logo of your webpage used in the home page [index.html]. If you must use the text content in the placement of your logo, then kindly change the logo on the homepage [index.html] to be the same to improve your design.

Screenshot from 2023-02-16 22-26-15

File: index.html [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+            <h2>Partners</h2>
+            <div class="partners">
+                <ul>
+                    <li><img src="/images/Exxon-Mobil.jpg" alt=""></li>
+                    <li><img src="/images/Massy-logo.jpg" alt=""></li>
+                    <li><img src="/images/Banks-DIH.jpg" alt=""></li>
+                    <li><img src="/images/Ministry-Of-Tourism.jpg" alt=""></li>
+                </ul>
+            </div>
+        </div>
```
Comment (zieeco) [inline index.html]: - Hey, Please, kindly note that this part of the webpage is a standalone section. it's a section of the partners of the page and it is not part of the footer. please, move it out from the footer and make it a section and not a div
   - Let's use the correct background color
   - The correct color of  the text on the logo
   - Add the missing red horizontal line under the word `Partner`
   - Let's use the correct spacing between the first and last logo
   - see the image below to compare your design

*your design*
Screenshot from 2023-02-16 22-40-00
*original design*
Screenshot from 2023-02-16 22-39-30

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,293 @@
...
+}
+
+.featured {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  padding: 20px 0;
+  padding-bottom: 50px;
+}
```
Comment (zieeco) [inline css/style.css]: - Let's kindly improve our style by making some adjustments to the spacing at the left and right of this section. see the images below:


*original design*
Screenshot from 2023-02-16 22-44-41

*your design*
Screenshot from 2023-02-16 22-43-59

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,293 @@
...
+  font-size: 20px;
+  margin-top: 1%;
+}
+
+.main-program {
+  background-image: url("/images/Guyana-1823-Monument.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: auto;
+}
```
Comment (zieeco) [inline css/style.css]: - Hey, please, let's improve our style to match the design as much as we can below:
   - Use the correct background color on the cards and the main background 
   - Align the program correctly on the desktop version
   - Add the missing white border hover effect on each card
   - see the images below to compare your design


*original design*
Screenshot from 2023-02-16 22-51-23


*your design*
Screenshot from 2023-02-16 22-58-38

File: index.html (hunk 10 above)
Comment (zieeco) [reply index.html]: ### MISSING FOOTER ON THE MOBILE VERSION OF THE HOMEPAGE


- Please, kindly make the footer visible in the mobile version of the homepage

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ClintonYounge ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Hi @ClintonYounge,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**